tests/unit/Claim/ClaimsTest.php

```diff
@@ -565,12 +568,6 @@ public function provideGetByRank() {
 				Statement::RANK_PREFERRED,
 				new Claims( array( $s2 ) ),
 			),
-			// s2 has RANK_PREFERRED, so doesn't match RANK_TRUTH
-			array(
-				new Claims( array( $s2 ) ),
-				Claim::RANK_TRUTH,
```

Comment:
`Claim::RANK_TRUTH` is a rank claims had, and only claims can have. It has no meaning if you do not have claims